pkg/fiber-middleware/oapi_validate_test.go

```diff
@@ -33,25 +33,36 @@ func doGet(t *testing.T, app *fiber.App, rawURL string) (*http.Response, error)
 	req.Header.Add("Accept", "application/json")
 	req.Host = u.Host
 
-	return app.Test(req)
+	r, err := app.Test(req)
+	if err != nil {
+		t.Fatalf("Failed to test request: %s", rawURL)
```

Comment:
This should also log the `err`:

```suggestion
		t.Fatalf("Failed to test request, URL=%q: %v", rawURL, err)
```